### DISCUSSION TITLE
Labs always gives a one line summary

### DISCUSSION THREAD

**Discussion by haveanicedavid** [community/community#18075] [Copilot Conversations]
Having a hard time figuring out where to submit feedback, so please direct me to the proper place if not here - I just installed the Copilot Labs extension, and notice it's always returning just one line responses.

For example, when I put in the example given on the homepage, this is my response:
<img width="563" alt="CleanShot 2022-06-07 at 22 30 58@2x" src="https://user-images.githubusercontent.com/9052511/172532098-bd9a1672-9498-44b5-a855-b4a277ca5cb7.png">

Is it possible I have something configured incorrectly? If not, is there any data you need to troubleshoot?

**Comment by eaftan** [comment]
Sorry for this, but this has been fixed now. If you haven't already, please update the Labs extension and let us know whether the problem is still occurring.
